### ISSUE TITLE
Inference graph file

### ISSUE THREAD

**Issue by prakashiiit** [huanzhang12/tensorflow-densenet-models#2]
I have used your github repo as an reference to my task of image Classification. I have trained Densenet 121 and got the ckpt file for the same for my dataset. Now further to get the result I need graph.pb file to do the inferencing. For the result I need graph.pb file, But I am unable to generate the graph file. Please help asap.

**Comment by prakashiiit** [comment]
Edit : I have got the graph.pb file. But after that how to freeze that graph ?? When I am trying to freez that graph following error is coming. No. of classes for me is 3.

`InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [1000] rhs shape= [3]
         [[Node: save/Assign_589 = Assign[T=DT_FLOAT, _class=["loc:@densenet121/logits/biases"], use_locking=true, validate_shape=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](densenet121/logits/biases, save/RestoreV2/_1179)]]
         [[Node: save/RestoreV2/_1188 = _Send[T=DT_FLOAT, client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_1194_save/RestoreV2", _device="/job:localhost/replica:0/task:0/device:CPU:0"](save/RestoreV2:594)]]`

Please help asap.
